src/lib/db/clients/index.js

```diff
@@ -93,5 +101,6 @@ export default {
   sqlite,
   cassandra,
   redshift: postgresql,
-  mariadb: mysql
+  mariadb: mysql,
+  cockroachdb: cockroachdb
```

Comment:
@rafiss here this should be `cockroachdb: postgresql`

Reply:
thanks for this catch! guess i had my labels backwards from here and the below comment.